app/src/main/res/values/styles.xml

```diff
@@ -6,6 +6,7 @@
         <item name="colorPrimary">@color/colorPrimary</item>
         <item name="colorPrimaryDark">@color/colorPrimaryDark</item>
         <item name="colorAccent">@color/colorAccent</item>
+        <item name="colorOnSecondary">@android:color/white</item>
```

Comment:
Just this should be enough

Reply:
But then the white FABs used in event cards have white icons making the icons invisible